dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -1148,6 +1149,177 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
     }
   }
 
+  override def listDLCs(): Future[Vector[DLCStatus]] = {
+    for {
+      ids <- dlcDAO.findAll().map(_.map(_.paramHash))
+      dlcFs = ids.map(findDLC)
+      dlcs <- Future.sequence(dlcFs)
+    } yield {
+      dlcs.collect {
+        case Some(dlc) => dlc
+      }
+    }
+  }
+
+  def findDLC(paramHash: Sha256DigestBE): Future[Option[DLCStatus]] = {
```

Comment:
any reason not to add this to `AnyDLCHDWalletApi`?

Reply:
`AnyDLCHDWalletApi extends DLCWalletApi`